presto-product-tests/src/main/java/io/prestosql/tests/hive/TestCsv.java

```diff
@@ -164,6 +216,103 @@ private void testCreatePartitionedCsvTableAs(String tableName, String additional
         query("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCreateCsvFileSkipHeader()
+    {
+        onHive().executeQuery("DROP TABLE IF EXISTS test_create_csvfile_skip_header");
+        onPresto().executeQuery("" +
+                "CREATE TABLE test_create_csvfile_skip_header" +
+                " (column1 varchar, column2 varchar, column3 varchar) " +
+                "WITH ( " +
+                "   format = 'CSV', " +
+                "   external_location = 'hdfs://hadoop-master:9000/user/hive/warehouse/TestCsvFileHiveTable/csv_table_with_header', " +
+                "   csv_separator= ','," +
```

Comment:
can you test different separator like `%`? So we could see that it works with different separators.